### PR TITLE
[games-action/supertuxkart] version bump to 0.9.2

### PR DESCRIPTION
**Pull request by nado**
Thanks to @ignatenko for his patch.

Stk team wont update angelscript for now because of a clang issue (see https://github.com/supertuxkart/stk-code/pull/2525#issuecomment-228756578), and they need to keep the clang support for OSX.

This patch modifies stk to be compatible with angelscript latest version (used in gentoo official overlay), instead of bundling angelscript again.

### CODE REVIEW

File: games-action/supertuxkart/files/supertuxkart-0.9.2-angelscript-compatibility.patch [hunk 1]
```diff
@@ -0,0 +1,325 @@
+From 6289b5fc298ac4bb48f426e3325a9bf514a603f1 Mon Sep 17 00:00:00 2001
+From: Igor Gnatenko <ignatenko@redhat.com>
+Date: Tue, 21 Jun 2016 11:48:13 +0200
+Subject: [PATCH] make code compatible with angelscript 2.31.0
```
Comment (hasufell) [inline games-action/supertuxkart/files/supertuxkart-0.9.2-angelscript-compatibility.patch]: supertuxkart is a rather friendly upstream, has this patch been communicated with upstream?

File: games-action/supertuxkart/supertuxkart-0.9.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2015 Gentoo Foundation
```
Comment (hasufell) [inline games-action/supertuxkart/supertuxkart-0.9.2.ebuild]: fix copyright

File: games-action/supertuxkart/supertuxkart-0.9.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (hasufell) [inline games-action/supertuxkart/supertuxkart-0.9.2.ebuild]: we should probably switch to EAPI=6

**Comment by hasufell** [comment]
does this build cleanly with EAPI=6? I didn't test

**Comment by nado** [comment]
Oh, I tested the fetch only, I will test it up to the merge if it has a chance of failing. Sorry for that.

**Comment by nado** [comment]
Here is a compliant EAPI=6 ebuild, took a while to understand that eapply_user.

**Pull request merged**